models/prices/base/prices_base_tokens.sql

```diff
@@ -29,7 +29,40 @@ FROM
     ('usdc-usd-coin', 'base', 'USDC', 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913, 6),
     ('wsteth-wrapped-liquid-staked-ether-20', 'base', 'wstETH',0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452,18),
     ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18),
-    ('degen-degen-base', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed,18),
-    ('dai-plus-overnight', 'base', 'DAI+', 0x65a2508c429a6078a7bc2f7df81ab575bd9d9275,18),
-    ('usd-plus-overnight', 'base', 'USD+', 0xb79dd08ea68a908a97220c76d19a6aa9cbde4376,18)
+    ('toshi-toshi', 'base', 'TOSHI', 0xac1bd2486aaf3b5c0fc3fd868558b082a531b2b4, 18),
+    ('degen-degen-base', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed, 18),
+    ('doginme-doginme', 'base', 'DOGINME', 0x6921B130D297cc43754afba22e5EAc0FBf8Db75b, 18),
+    ('prime-echelon-prime', 'base', 'PRIME', 0xfa980ced6895ac314e7de34ef1bfae90a5add21b, 18),
+    ('rsr-reserve-rights', 'base', 'RSR', 0xab36452dbac151be02b16ca17d8919826072f64a, 18),
+    ('sdex-smardex', 'base', 'SDEX', 0xfd4330b0312fdeec6d4225075b82e00493ff2e3f, 18),
+    ('tbtc-tbtc', 'base', 'TBTC', 0x236aa50979d5f3de3bd1eeb40e81137f22ab794b, 18),
+    ('mbs-monkeyleague', 'base', 'MBS', 0x8Fbd0648971d56f1f2c35Fa075Ff5Bc75fb0e39D, 18),
+    ('isk-iskra-token', 'base', 'ISK', 0x17d2628d30f8e9e966c9ba831c9b9b01ea8ea75c, 18),
+    ('vela-vela-token', 'base', 'VELA', 0x5a76a56ad937335168b30df3aa1327277421c6ae, 18),
+    ('tower-tower', 'base', 'TOWER', 0xf7C1CEfCf7E1dd8161e00099facD3E1Db9e528ee, 18),
+    ('frame-frametoken', 'base', 'FRAME', 0x91F45aa2BdE7393e0AF1CC674FFE75d746b93567, 18),
+    ('omni-omnicat', 'base', 'OMNI', 0xc48e605c7b722a57277e087a6170b9e227e5ac0a, 18),
+    ('bsx-basex', 'base', 'BSX', 0xd5046b976188eb40f6de40fb527f89c05b323385, 18),
+    ('fxdx-fxdx', 'base', 'FXDX', 0x30b593f8c3ab37615359b4e0e6df2e06d55bb55d, 18),
```

Comment:
note this is returning a failure for invalid id:
https://github.com/duneanalytics/spellbook/actions/runs/7918639192/job/21617536821?pr=5353#step:6:47

Reply:
Looks like it should be `fxdx-fxdx-exchange` as the ticker